python/ray/_private/structured_logging/constants.py

```diff
@@ -0,0 +1,47 @@
+from enum import Enum
+
+# A set containing the standard attributes of a LogRecord. This is used to
+# help us determine which attributes constitute Ray or user-provided context.
+# http://docs.python.org/library/logging.html#logrecord-attributes
+LOGRECORD_STANDARD_ATTRS = {
+    "args",
+    "asctime",
+    "created",
+    "exc_info",
+    "exc_text",
+    "filename",
+    "funcName",
+    "levelname",
+    "levelno",
+    "lineno",
+    "message",
+    "module",
+    "msecs",
+    "msg",
+    "name",
+    "pathname",
+    "process",
+    "processName",
+    "relativeCreated",
+    "stack_info",
+    "thread",
+    "threadName",
+    "taskName",
+}
+
+
+class LogKey(str, Enum):
+    # Core context
+    JOB_ID = "job_id"
+    WORKER_ID = "worker_id"
+    NODE_ID = "node_id"
+    ACTOR_ID = "actor_id"
+    TASK_ID = "task_id"
+
+    # Logger built-in context
+    TS = "ts"
+    LEVEL = "level"
+    MSG = "msg"
+    FILENAME = "filename"
+    LINENO = "lineno"
+    EXC_TEXT = "exc_text"
```

Comment:
Let's rename this to "exception"

Reply:
Is it acceptable to use a different naming style than that of the Python standard library?

Reply:
I may prefer to keep using `EXC_TEXT` because it's hard for developers to understand that `EXCEPTION` refers to `EXC_TEXT` rather than `EXC_INFO`, which is another built-in attribute of LogRecord, based on the variable name alone.

Reply:
sync with @jjyao offline: We decide to follow the naming convention of the standard logging library, so we also rename `TS` to `ASCTIME`, `LEVEL` to `LEVELNAME`, and `MSG` to `MESSAGE`. Note that LogRecord has both `msg` and `message`, but we want to use `message` instead of `msg`.

Reply:
@kevin85421 just want to bring your attention, "message" only exist when the formatter is called. "msg" is the original message for the log record. Serve perviously has this bug that uses message directly not aware it might not exist and produced structured logging with all the attributes but no message.

Reply:
but looking at your latest commit that seems like not relevant 😛

Reply:
> just want to bring your attention, "message" only exist when the formatter is called. "msg" is the original message for the log record.

I think it is fine to use `message` here. In `logging` source code, it calls `record.message = record.getMessage()` to convert `msg` to `message` ([code](https://github.com/python/cpython/blob/cb6f75a32ca2649c6cc1cabb0301eb783efbd55b/Lib/logging/__init__.py#L708)). This PR's formatter calls `getMessage` directly.